.pylint.rc

```diff
@@ -0,0 +1,58 @@
+[MASTER]
+jobs=4
+persistent=yes
+suggestion-mode=yes
+unsafe-load-any-extension=no
+
+# Blacklist files or directories (basenames, not paths)
+ignore=
+
+# blacklist files or directories by regex  (basenames, not paths)
+ignore-patterns=
+
+[EXCEPTIONS]
+
+overgeneral-exceptions=Exception,RaidenUnrecoverableError
```

Comment:
this is new, `RaidenUnrecoverableError` was not part of the previous config [ref](https://pylint.readthedocs.io/en/latest/technical_reference/features.html?highlight=overgeneral%20exceptions).

> Exceptions that will emit a warning when being caught. Defaults to "Exception".

The unrecoverable exceptions should not be caught (sadly this will not cover inheritance)